modules/deleting-red-hat-log-visualization.adoc

```diff
@@ -0,0 +1,29 @@
+:_newdoc-version: 2.18.4
+:_template-generated: 2025-06-03
+:_mod-docs-content-type: PROCEDURE
+
+[id="deleting-red-hat-log-visualization_{context}"]
+= Deleting Red{nbsp}Hat log visualization
+
+When updating from Logging 5 to Logging 6, you must delete Red{nbsp}Hat Log Visualization before installing the UIPlugin.
+
+.Prerequisites
+* You have administrator permissions.
+* You installed the {oc-first}.
+
+.Procedure
+
+. Run the following command:
++
+[source,terminal]
+----
+$ oc get consoles.operator.openshift.io -o yaml -o jsonpath='{.spec.plugins}' |grep "logging-view-plugin" && oc patch consoles.operator.openshift.io/cluster  --type json -p='[{"op": "remove", "path": "/spec/plugins", "value":{'logging-view-plugin'}}]' 
```

Comment:
We needn't patch consoles.operator.openshift.io/cluster.   The console-operator will delete it automatically once "oc delete consoleplugins logging-view-plugin"

Reply:
done